tests/test_dcrModel.py

```diff
@@ -195,35 +214,137 @@ def testDcrCalculation(self):
         afwImage.utils.defineFilter("gTest", self.lambdaEff,
                                     lambdaMin=self.lambdaMin, lambdaMax=self.lambdaMax)
         filterInfo = afwImage.Filter("gTest")
-        rotAngle = 0.*radians
+        rotAngle = 0.*degrees
         azimuth = 30.*degrees
         elevation = 65.*degrees
         pixelScale = 0.2*arcseconds
         visitInfo = self.makeDummyVisitInfo(azimuth, elevation)
         wcs = self.makeDummyWcs(rotAngle, pixelScale, crval=visitInfo.getBoresightRaDec())
         dcrShift = calculateDcr(visitInfo, wcs, filterInfo, dcrNumSubfilters)
-        refShift = [afwGeom.Extent2D(-0.5363512808, -0.3103517169),
-                    afwGeom.Extent2D(0.001887293861, 0.001092054612),
-                    afwGeom.Extent2D(0.3886592703, 0.2248919247)]
+        refShift = [afwGeom.Extent2D(-0.3103517169, -0.5363512808),
+                    afwGeom.Extent2D(0.001092054612, 0.001887293861),
+                    afwGeom.Extent2D(0.2248919247, 0.3886592703)]
         for shiftOld, shiftNew in zip(refShift, dcrShift):
             self.assertFloatsAlmostEqual(shiftOld.getX(), shiftNew.getX(), rtol=1e-6, atol=1e-8)
             self.assertFloatsAlmostEqual(shiftOld.getY(), shiftNew.getY(), rtol=1e-6, atol=1e-8)
 
+    def testDcrSubfilterOrder(self):
+        """Test that the bluest subfilter always has the largest amplitude.
+        """
+        seed = 6
+        rng = np.random.RandomState(seed)
+        dcrNumSubfilters = 3
+        afwImage.utils.defineFilter("gTest", self.lambdaEff,
+                                    lambdaMin=self.lambdaMin, lambdaMax=self.lambdaMax)
+        filterInfo = afwImage.Filter("gTest")
+        pixelScale = 0.2*arcseconds
+        rotAngle = 360.*rng.rand()*degrees
+        azimuth = 360.*rng.rand()*degrees
+        elevation = (45. + rng.rand()*40.)*degrees  # Restrict to 45 < elevation < 85 degrees
```

Comment:
Given that you set a random seed, this will always have the same value. Maybe lift the internals into a `checkDcrSubFilterOrder` method, and have a loop over some carefully chosen values (including zenith, north, south, east, west) in the `testDcrSub...` method that call `check...`?